projects/ion/src/lib/dropdown/dropdown.component.spec.ts

```diff
@@ -212,7 +167,7 @@ describe('IonDropdownComponent / Disabled', () => {
   });
 });
 
-describe('IonDropdownComponent / Multiple', () => {
+describe('IonDropdownComponent / Multiple / Clear Options', () => {
```

Comment:
Could you refactor these tests cases to separate the cases that use `defaultMultiple` on sut from the cases that use `multipleWithInitialValue`?

Reply:
Sure, but i need more informations, i will talk with about it